seacrowd/sea_datasets/id_vaccines_tweets/id_vaccines_tweets.py

```diff
@@ -0,0 +1,124 @@
+import datasets
+import pandas
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (DEFAULT_SEACROWD_VIEW_NAME,
+                                      DEFAULT_SOURCE_VIEW_NAME, Tasks, Licenses)
+
+_DATASETNAME = "id_vaccines_tweets"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
+
+_LANGUAGES = ["ind"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+_CITATION = """\
```

Comment:
Could you please adjust the formatting in `_CITATION`? For example, like the `_CITATION` in [this dataloader](https://github.com/SEACrowd/seacrowd-datahub/blob/master/seacrowd/sea_datasets/id_google_play_review/id_google_play_review.py).